### ISSUE TITLE
Observing "_df" does not trigger when a cell is changed

### ISSUE THREAD

**Issue by n4562** [quantopian/qgrid#170]
Example:
`qgrid_widget.observe(func, names=['_df'])`
will not trigger on a 'cell_change' (i.e. double clicking a cell and changing the value).

Hack that works ('touch'):
`df = self._df.copy(); self._df = df`

**Comment by TimShawver** [comment]
Thanks, I have a fix for this and I'll include it in the next release.

**Comment by TimShawver** [comment]
This should be fixed in version 1.0.1 which has now been released.  The change event for the `_df` traitlet should now fire for edits, sort changes, and filter changes. Thanks again for reporting this and feel free to reopen if the fix doesn't work for you.

**Comment by rhsmits91** [comment]
I am having the same problem in version 1.1.1: 

```
import pandas as pd
import qgrid
import ipywidgets as widgets

df = pd.DataFrame(pd.np.arange(32).reshape(16, 2), columns=list('AB'))
grid = qgrid.show_grid(df)
output = widgets.Output()

check = []

@output.capture(clear_output=True)
def apply_changes(change):
    grid.add_row([('A', 100), ('B', 200)])
    check.append('add')
    
grid.observe(apply_changes, names=['_df'])
```

then when I change a row value, no row gets added nor does 'add' get added to `check`

any ideas why this might be?
